src/core/renderers/webgl/WebGLRenderer.js

```diff
@@ -475,6 +475,12 @@ WebGLRenderer.prototype.destroy = function (removeView)
     this.view.removeEventListener('webglcontextlost', this.handleContextLost);
     this.view.removeEventListener('webglcontextrestored', this.handleContextRestored);
 
+    for (var key in utils.BaseTextureCache) {
+        var texture = utils.BaseTextureCache[key];
+        texture.off('update', this.updateTexture, this);
+        texture.off('dispose', this.destroyTexture, this);
+    }
```

Comment:
This is a fantastic change, but just want to point out that it is not garuenteed to capture all textures that this renderer put a listener on. You can create textures without them being put into that cache, and in those cases they won't be caught by this. But this is much better than not having it at all.